packages/testing/src/rawtx.ts

```diff
@@ -40,6 +40,19 @@ export async function createSignedTxnHex (
   return signed.hex
 }
 
+/**
+ * Get fixture for a txid
+ *
+ * @param {MasterNodeRegTestContainer} container
+ * @param {string} txid
+ * @return {Promise<string>} Fixture as hex string
+ */
+export async function getFixture (container: MasterNodeRegTestContainer, txid: string): Promise<string> {
```

Comment:
Too ambiguous, should rename to exactly what it does

Reply:
Agree